server/etcdmain/config.go

```diff
@@ -276,6 +276,10 @@ func newConfig() *config {
 	// gateway
 	fs.BoolVar(&cfg.ec.EnableGRPCGateway, "enable-grpc-gateway", cfg.ec.EnableGRPCGateway, "Enable GRPC gateway.")
 
+	// grpc dump & replay
```

Comment:
We should be very explicit that enabling this is exposing a 'security treat'
that any access to the port can provide cross-user access to the payloads. 

If we do this using expicit grpc API, we could/should restrict this feature to 'admins' only.